content/blog/2019/root-transition/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: User guide for extenting Istio root certificate lifetime
+description: The user guide to extend Istio root certificate lifetime.
+publishdate: 2019-06-06
+subtitle:
+attribution: Oliver Liu
+twitter:
+keywords: [security, PKI, certificate, Citadel]
+
+---
+
+## Background
```

Comment:
Add something like "If you are not currently using and have no plan to use Istio security features, e.g., mutual TLS, RBAC, etc, in the future, your deployment is not affected and you can skip this announcement."

Reply:
Isn't that covered in scenarios?